pydicom/pixel_data_handlers/pylibjpeg_handler.pyi

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 pydicom authors. See LICENSE file for details.
+
+import numpy as np
+
+from pydicom.dataset import Dataset
+from pydicom.uid import UID
+
+
+def is_available() -> bool: ...
+
```

Comment:
Question: is the pyi file format widely supported?  Should these be better put in via Python 3.6+ typing directly in the files?

Reply:
>One query re the added .dcm files - should they now go to the zenodo data store?

Sure, how do I go about that?

>Question: is the pyi file format widely supported? Should these be better put in via Python 3.6+ typing directly in the files?

I was getting a circular import when I did that, hence the *.pyi [stub file](https://www.python.org/dev/peps/pep-0484/#stub-files). The alternative is to do something like this:
```python
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from pydicom.dataset import Dataset
```

Do you have a preference? I'm starting to lean towards the second one now that I look at it again

Reply:
Actually, a little bit more reading says I can do this:
```python
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from pydicom.dataset import Dataset

def as_array(ds: "Dataset") -> np.ndarray:
    # code goes here
```

Reply:
Changed to remove stub file

Reply:
> Sure, how do I go about that?

I think I missed replying to this ... I don't personally know ... @SimonBiggs, can you save us the trouble of researching how to do this?  I noticed you had 'Future test files will likely be their own Zenodo submission' in the description, but I assume we have to upload to same DOI number somehow?  I also see that 'This DOI represents all versions, and will always resolve to the latest one' on the side bar, but I don't see any options to upload anywhere on the page.

Reply:
Essentially all that is needed is to upload a new submission to Zenodo and then to add the download URL to the "urls.json" file. Then, use the API to download the file, and the first download will result in adding the hash to the hash.json file.

Reply:
I would also argue it's likely not worth the effort for files < 100 kB in size...

Reply:
> I would also argue it's likely not worth the effort for files < 100 kB in size...

Yeah, I agree with that... didn't look at the size in this case.

Reply:
`US1_UNCI.dcm` is larger though, and it's been moved...

Reply:
Is it possible that we put the large datasets in a branch on their own and create a workflow so that when the branch is updated it automatically updates Zenodo?

Reply:
I've been reading up on this - There is supposed [to be a way](https://help.zenodo.org/#versioning) to continue changing datasets, and having one DOI that automatically references the latest one.

However, I don't see any way to upload in the UI, but there is a [REST API](https://developers.zenodo.org/#quickstart-upload), perhaps that can be used somehow.

Maybe only the creator of a dataset can upload new files?  If needed I can set up a new one under my name so I can add files in future.

Reply:
Speaking without any knowledge of the subject, can we have a branch just for the data and create a workflow that automatically propagates changes to zenodo? 

I am very lazy 😁

Reply:
I see it is possible to make a new version: https://youtu.be/S1qK_TA52e4?t=363, but it is not clear that you can add new files (the example shows modifying an existing file with a new upload).  Plus it does seem like only the original author would have that ability to make the new version.  I'll check into making my own dataset and see how it goes...

Reply:
> can we have a branch just for the data and create a workflow that automatically propagates changes to zenodo?

I was wondering whether we can just make it as its own repo, in which case, just as pydicom gets automatically published, so would that with new "releases" for that repo.  It might not be as easy to access files then, though, because I think the whole thing is always zipped for repos (...?)